src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: (num) => result + num,
+    subtract: (num) => result - num,
+    multiply: (num) => result * num,
+    divide: (num) => (num === 0 ? result : result / num),
```

Comment:
Consider throwing an error or returning a specific value when attempting to divide by zero, as returning the current result might be misleading.